SIT.Manager.Avalonia/ViewModels/ToolsPageViewModel.cs

```diff
@@ -217,4 +236,115 @@ private async Task ClearCache()
             }
         }
     }
+
+    private async Task GenerateDiagnosticReport()
+    {
+        var results = await new SelectLogsDialog().ShowAsync();
+        List<Tuple<string, string>> DiagnosticData = new(4);
+
+        if (results.IncludeClientLog)
+        {
+            string eftLogPath = EFTLogPath;
+            DiagnosticData.Add(new(Path.GetFileName(eftLogPath), await File.ReadAllTextAsync(eftLogPath)));
+        }
```

Comment:
Add a check to see if the file actually exists before you try and read it otherwise the app will close